alchemist/alchemist-fxui/src/main/kotlin/it/unibo/alchemist/boundary/jfx/events/mouse/BasicMouseTriggerAction.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2010-2020, Danilo Pianini and contributors
+ * listed in the main project's alchemist/build.gradle.kts file.
+ *
+ * This file is part of Alchemist, and is distributed under the terms of the
+ * GNU General Public License, with a linking exception,
+ * as described in the file LICENSE in the Alchemist distribution's top directory.
+ */
+
+package it.unibo.alchemist.boundary.jfx.events.mouse
+
+/**
+ * Simple mouse actions.
+ */
+enum class BasicMouseTriggerAction : MouseTriggerAction {
```

Comment:
Why an enum with a single value? It's usually used to produce singletons, but you could object instead.